src/pages/ConnectWallet/components/WalletList.tsx

```diff
@@ -193,7 +194,7 @@ export const MobileWalletList: React.FC<MobileWalletDialogProps> = ({
     isInitializingWallet,
   ])
 
-  return isLoading ? (
+  return isLoading || state.isLoadingLocalWallet ? (
```

Comment:
Probably the most important change, load things if we are currently loading the local wallet